app/main/session_parser.py

```diff
@@ -236,38 +236,35 @@ def restore_active_brew_sessions():
             # print('DEBUG: restore_active_sessions() found {} as an active session'.format(file))
             brew_session = load_brew_session(file)
             # print('DEBUG: restore_active_sessions() {}'.format(brew_session))
-            if brew_session['uid'] not in active_brew_sessions:
-                active_brew_sessions[brew_session['uid']] = []
+            uid = brew_session['uid']
+            if uid not in active_brew_sessions:
+                session = PicoBrewSession()
 
-            session = PicoBrewSession()
+            session = active_brew_sessions[uid]
             session.file = open(file, 'a')
             session.file.flush()
             session.filepath = file
-            session.alias = brew_session['alias']
             session.created_at = brew_session['date']
             session.name = brew_session['name']
             session.type = brew_session['type']
             session.session = brew_session['session']                   # session guid
             session.id = -1                                             # session id (integer)
 
-            if 'machine_type' in brew_session:
-                session.machine_type = brew_session['machine_type']     # keep track of machine type of uid (if known)
-
             if 'recovery' in brew_session:
                 session.recovery = brew_session['recovery']             # find last step name
 
             # session.remaining_time = None
-            session.is_pico = brew_session['is_pico']
```

Comment:
'is_pico', 'alias', and 'machine_type' aren't persisted into the session file, maybe they should be?? but we don't really need that data for anything programmatically except for the photo... so in theory the icon could go missing again if you restart the server without aliasing the device... no biggie